.gitignore

```diff
@@ -25,3 +25,9 @@ recommendationspb
 
 # IDE
 .vscode
+
+# Python
+__pycache__
+
+# Unit tests
+.coverage
```

Comment:
Editor config

Reply:
Yeah i wasn't sure but it's getting fixed